core/src/main/java/org/apache/iceberg/MergingSnapshotProducer.java

```diff
@@ -60,7 +60,7 @@
 
   private final String tableName;
   private final TableOperations ops;
-  private final PartitionSpec spec;
+  private PartitionSpec spec;
```

Comment:
Can you move this into the "update data" section? These fields are all final.

Reply:
Done.